### PR TITLE
Feature/application portal

### PR DESCRIPTION
**Pull request by krishnakpandian**
Problem
=======
Application Portal 

Solution
========
What I/we did to solve this problem
* Adding all the Logic and Files to support the application system

Change Summary:
---------------
* Updated Storage to utilize Session Storage
* Added Props for application 
* Updated Styles to include mixins for application Pages
* Created Application Pages + Application Context
* Created Application Input Components

Dev-Ops
=======
If you added an ENV variable, please check off that you've updated the following  
- [ ] Key & Sample value to `.env` & `sample.env` 
- [ ] GCP Secret Manager (Both development and production)
- [ ] Github Secrets (Added a development and production variable)
- [ ] Github Workflows `.github/workflows/dev.yml` & `.github/workflows/prod.yml`
- [ ] webpack-config.js

Images/Important Notes (optional):
-----------------------
* Insert any notes/issues, dependencies added or removed, or images

### CODE REVIEW

File: src/views/Portal/components/ApplicationStatus/index.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  border-radius: 18px;
+
+  @include responsiveness.screen(smallest, phone, tablet) {
+    width: 75%;
+    padding: 10px;
+  }
+  &__button {
+    background: #A5D6D9;
+    border-radius: 5px;
+    font-family: "Lato", "sans-serif";
```
Comment (w-gao) [inline src/views/Portal/components/ApplicationStatus/index.scss]:
```suggestion
    font-family: "Lato", sans-serif;
```

File: src/views/Portal/components/ApplicationStatus/index.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    font-style: normal;
+    font-weight: normal;
+    @include responsiveness.font($responsive: 4.1vw, $min: 18px, $max: 25px);
+    line-height: 30px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #000000;
```
Comment (w-gao) [inline src/views/Portal/components/ApplicationStatus/index.scss:29]: Can we add a `cursor: pointer;` here?

**Review by w-gao — APPROVED** [review]
Super excited to get this in!

File: src/views/Portal/components/ApplicationStatus/index.scss (hunk 2 above)
Comment (Alperdec) [reply src/views/Portal/components/ApplicationStatus/index.scss:29]: Sure!
Good catch

**Pull request merged**